graylog2-web-interface/src/routing/hooks/useShowRouteForEntity.ts

```diff
@@ -68,6 +67,8 @@ const useShowRouteForEntity = (id: string, type: string) => {
       return Routes.stream_search(id);
     case 'search_filter':
       return Routes.getPluginRoute('MY-FILTERS_DETAILS_FILTERID')?.(id);
+    case 'sigma_rule':
```

Comment:
This adds Enterprise types into open code. We should avoid that.

Reply:
Hello @bernd,

I will create a follow-up issue to remove this and the existing enterprise types that are currently in the hook.

Reply:
Thanks! :+1: